ert_shared/cli/workflow.py

```diff
@@ -4,15 +4,16 @@
 
 
 def execute_workflow(workflow_name):
+    logger = logging.getLogger(__name__)
     workflow_list = ERT.ert.getWorkflowList()
     try:
         workflow = workflow_list[workflow_name]
     except KeyError:
         msg = "Workflow {} is not in the list of available workflows"
-        logging.error(msg.format(workflow_name))
+        logger.error(msg.format(workflow_name))
         return
     context = workflow_list.getContext()
     workflow.run(ert=ERT.ert, verbose=True, context=context)
     all_successfull = all([v["completed"] for k, v in workflow.getJobsReport().items()])
     if all_successfull:
-        logging.info("Workflow {} ran successfully!".format(workflow_name))
+        logger.info("Workflow {} ran successfully!".format(workflow_name))
```

Comment:
Convert to `f` string while it is being changed?

Reply:
Just a note: formatting when logging has performance considerations, and for most cases it's recommended to use for example `%s` in your string and pass the values to log function. See: https://docs.python.org/3/howto/logging.html#optimization

Reply:
It is the recommended practice, but that generally applies to computationally expensive operations. Think formatted string should be ok in this case (without having actually done a benchmark)